src/pages/organizations/[organizationid].js

```diff
@@ -110,14 +110,17 @@ export default function Organization(props) {
       // manipulate the map
       const { map } = mapContext;
       if (map && organization) {
-        // map.flyTo(tree.lat, tree.lon, 16);
-        map.setFilters({
-          map_name: organization.name,
-        });
         // TODO why I must try/catch this?
         try {
-          await map.loadInitialView();
-          map.rerender();
+          if (router.query.bounds) {
+            await map.goToBounds(router.query.bounds);
+          } else {
+            map.setFilters({
+              map_name: organization.name,
+            });
+            await map.loadInitialView();
+            map.rerender();
+          }
```

Comment:
@Mohmn setFilter will reset the state of the map, to load expected map, like: load organization map above, and even we got bounds from query parameter, I think we should stop to load the org map? 

According to your code, we just jump to the map area, and without loading the correct state of the map? (by setFilter)

Reply:
You can test it in this way: 
1. Open an planter map: https://dev-k8s.treetracker.org/web-map-beta/demo/planters/940
2. Then move the map a little bit, so the `bounds` shows up in url
3. Copy the URL and open in browser, then the result should be:
   1. The map jump to the bounds
   2. The map show the planters tree/clusters.